kivy/data/style.kv

```diff
@@ -569,6 +569,8 @@
     size_hint_x: None if not root.inside_group else 1
     width: [dp(48) if (root.icon and not root.inside_group) else max(dp(48), (self.texture_size[0] + dp(32))), self.size_hint_x][0]
     color: self.color[:3] + [0 if (root.icon and not root.inside_group) else 1]
+    padding_x: dp(8)
+    minimum_width: self.texture_size[0]
```

Comment:
Ah yes!  Well... they could using `minimum_width: self.minimum_width and dp(50)`, but I take your meaning.  This is a shortcoming of the current implementation of property constraints.  Maybe it can be fixed.  I'll look into it.